### PR TITLE
CSS `exp()` function - Add two examples, add return value and update description

### PR DESCRIPTION
**Pull request by ramiy**
### Description
@wbamberg requested to add  a couple of examples - https://github.com/mdn/content/pull/20826#pullrequestreview-1112765075

### Motivation
Should have been part of #20826 but I haven't finished testing the examples...

### Related issues and pull requests
Relates to #20826

### CODE REVIEW

File: files/en-us/web/css/exp/index.md [hunk 1]
```diff
@@ -17,34 +17,136 @@ browser-compat: css.types.exp
...
 
 ### Formal syntax
 
 {{CSSSyntax}}
 
+## Examples
+
+### Rotate elements
+
+The `exp()` function can be used to {{CSSxRef("transform-function/rotate", "rotate")}} elements as it return an {{CSSxRef("&lt;angle&gt;")}}.
```
Comment (wbamberg) [inline files/en-us/web/css/exp/index.md:57]: The "Return value" section says this returns a number, not an angle, and this example does not work for me in Safari.

**Review by wbamberg — COMMENTED** [review]
The "Rotate elements" example doesn't work for me in Safari, are you sure it is correct?

File: files/en-us/web/css/exp/index.md (hunk 1 above)
Comment (ramiy) [reply files/en-us/web/css/exp/index.md:57]: Sorry, copy&paste... It return a number not an angel. That is we have to multiply this number by 1turn.

File: files/en-us/web/css/exp/index.md [hunk 2]
```diff
@@ -17,34 +17,136 @@ browser-compat: css.types.exp
...
+```
+
+```css
+div.box {
+  width: 100px;
+  height: 100px;
+  background: linear-gradient(orange, red);
+}
+div.box-1 {
+  transform: rotate(1turn * exp(-1));    // 0.3678794411714423turn
```
Comment (wbamberg) [inline files/en-us/web/css/exp/index.md:88]: I don't think this is valid.

Comment (ramiy) [reply files/en-us/web/css/exp/index.md:88]: Why not valid? When using negative values, `exp()` returns fractions (between `0` to `0.9999...`). Passing `-Infinity` should return `0`.

Comment (wbamberg) [reply files/en-us/web/css/exp/index.md:88]: The example looks like this in Safari:

Screen Shot 2022-10-18 at 5 06 31 PM

... and it looks like Safari thinks that `rotate(1turn * 2)` (for example) is not valid:

Screen Shot 2022-10-18 at 5 08 23 PM

Comment (ramiy) [reply files/en-us/web/css/exp/index.md]: @wbamberg Found the issue. `rotate()` is not a math function. To calculate, we need a `calc()` function.

**Review by wbamberg — APPROVED** [review]
👍 thank you @ramiy !

**Pull request merged**